lib/formatter.js

```diff
@@ -1,6 +1,12 @@
-function format(paramValues, prefix, wrapInBrackets) {
+function format(paramValues, prefix) {
```

Comment:
Update https://github.com/kibertoad/objection-find-query/blob/master/test/formatter.spec.js to reflect removal of a parameter